tests/system_tests_autolinks.py

```diff
@@ -53,17 +53,18 @@ def setUpClass(cls):
             #
             # Create a pair of default auto-links for 'node.1'
             #
-            ('autoLink', {'addr': 'node.1', 'containerId': 'container.1', 'dir': 'in'}),
-            ('autoLink', {'addr': 'node.1', 'containerId': 'container.1', 'dir': 'out'}),
+            ('autoLink', {'addr': 'node.1', 'containerId': 'container.1', 'direction': 'in'}),
+            ('autoLink', {'addr': 'node.1', 'containerId': 'container.1', 'direction': 'out'}),
 
             #
             # Create a pair of auto-links on non-default phases for container-to-container transfers
             #
-            ('autoLink', {'addr': 'xfer.2', 'containerId': 'container.2', 'dir': 'in',  'phase': '4'}),
-            ('autoLink', {'addr': 'xfer.2', 'containerId': 'container.3', 'dir': 'out', 'phase': '4'}),
+            ('autoLink', {'addr': 'xfer.2', 'containerId': 'container.2', 'direction': 'in',  'phase': '4'}),
+            ('autoLink', {'addr': 'xfer.2', 'containerId': 'container.3', 'direction': 'out', 'phase': '4'}),
 
             #
             # Create a pair of auto-links with a different external address
+            # Leave the direction as dir to test backward compatibility.
```

Comment:
Is there a test that has both dir and direction specified to see that it fails?

Reply:
No there is not such a test. Thanks for pointing that out. I will add such a test to this PR shortly.